index.html

```diff
@@ -1277,6 +1277,15 @@
 privacy threats to a particular individual (see <a href="#principle-collective-privacy">collective
 privacy</a>). But some people might object anyway, in particular instances or in general.
 
+A [=user agent=] must act in the interest of the user when reporting aggregated data, and should
+not participate in sharing of aggregated data in the interests of any other [=actor=], even if
+the aggregation scheme is unlikely to reveal information about a specific user. For example, in
+web advertising, the user has an interest in receiving the maximum value in ad-supported content
+for the lowest cost in attention, risk and computing resources, while some [=actor=]s have an
+interest in maximizing ad revenue while minimizing the cost of content.  The user agent has an
+obligation to share data in a way that advances the user's interests, not those of the other
+actors.
```

Comment:
Maybe this rewording?

```suggestion
A [=user agent=] must act in the interest of its user when reporting aggregated data and should
not use the fact that data is aggregated as an excuse to share more data than is in that user's interest.
```